pkg/reaper/nodereaper/nodereaper.go

```diff
@@ -499,24 +502,24 @@ func (ctx *ReaperContext) reapOldNodes(w ReaperAwsAuth) error {
 			return nil
 		}
 
+		// Skip if target node is self
```

Comment:
move up because we don't have to do anything else if this is true

Reply:
Was this moved up already?

Reply:
no i moved it up in this PR.

Reply:
Ah I see, I read it as a TODO